sentry-samples/sentry-samples-spring-boot/src/main/java/io/sentry/samples/spring/boot/SentryDemoApplication.java

```diff
@@ -1,11 +1,13 @@
 package io.sentry.samples.spring.boot;
 
+import io.sentry.spring.EnableSentry;
```

Comment:
This must have accidentally sneaked in. Run `./gradlew spotlessApply` from the command line, to remove all unused imports, re-order imports, format code etc.

Reply:
Yup...my bad. lint quickly fixed this :)